tests/archives/cvt_archive_test.py

```diff
@@ -136,3 +136,40 @@ def test_add_single_without_overwrite(data, add_mode):
     assert np.isclose(value, low_objective - data.objective)
     assert_archive_elite(data.archive_with_elite, data.solution, data.objective,
                          data.measures, data.centroid, data.metadata)
+
+
+def test_chunked_calculation_short():
+    """Testing accuracy of chunked computation"""
+    archive = CVTArchive(solution_dim=0,
+                         cells=10,
+                         ranges=[(-1, 1), (-1, 1)],
+                         samples=10,
+                         chunk_size=1000,
```

Comment:
This isn't really testing the chunking because the chunk size is so large so everything just runs in one chunk -- I suggest setting it to something small like 2 so that you can force the chunking to run.